### PR TITLE
implements #219: Show all routing parameters

### PR DESCRIPTION
**Pull request by samsonasik**
@RalfEggert here we go. also fixes the zdt-toolbar-info-redundant class that only show last record.

### CODE REVIEW

**Comment by samsonasik** [comment]
Fixes #219

File: src/Collector/RequestCollector.php [hunk 1]
```diff
@@ -71,7 +71,10 @@ public function collect(MvcEvent $mvcEvent)
             'status'     => $mvcEvent->getResponse()->getStatusCode(),
             'route'      => ($match === null) ? 'N/A' : $match->getMatchedRouteName(),
             'action'     => ($match === null) ? 'N/A' : $match->getParam('action', 'N/A'),
-            'controller' => ($match === null) ? 'N/A' : $match->getParam('controller', 'N/A')
+            'controller' => ($match === null) ? 'N/A' : $match->getParam('controller', 'N/A'),
+            'other_route_parameters' => ($match === null) ? 'N/A' : array_filter($match->getParams(), function ($key) {
+                return $key !== 'action' && $key !== 'controller';
```
Comment (snapshotpl) [inline src/Collector/RequestCollector.php]: Usage of `in_array` should improve readability.

Comment (samsonasik) [inline src/Collector/RequestCollector.php]: implemented.

**Comment by weierophinney** [comment]
Thanks, @samsonasik

**Pull request merged**